.azure-pipelines/Linux-CI.yml

```diff
@@ -4,35 +4,45 @@ trigger:
 jobs:
 - job: 'Test'
   pool:
-    vmImage: 'Ubuntu-18.04'
+    vmImage: ubuntu-latest
   strategy:
     matrix:
+      py311-ml:
+        python.version: '3.11'
+        onnx_ml: true
+        onnx_debug: false
+        documentation: false
+      py310-ml:
+        python.version: '3.10'
+        onnx_ml: true
+        onnx_debug: false
+        documentation: false
       py39-ml-debug:
         python.version: '3.9'
-        onnx_ml: 1
-        onnx_debug: 1
-        documentation: 0
+        onnx_ml: true
+        onnx_debug: true
+        documentation: false
       py38:
         python.version: '3.8'
-        onnx_ml: 0
-        onnx_debug: 0
-        documentation: 0
+        onnx_ml: false
+        onnx_debug: false
+        documentation: false
       py38-ml:
         python.version: '3.8'
-        onnx_ml: 1
-        onnx_debug: 0
-        documentation: 1
+        onnx_ml: true
+        onnx_debug: false
+        documentation: true
       py37:
         python.version: '3.7'
-        onnx_ml: 0
-        onnx_debug: 0
-        documentation: 0
+        onnx_ml: false
+        onnx_debug: false
+        documentation: false
       py37-ml:
         python.version: '3.7'
-        onnx_ml: 1
-        onnx_debug: 0
-        documentation: 0
-    maxParallel: 6
+        onnx_ml: true
+        onnx_debug: false
+        documentation: false
+    maxParallel: 7
```

Comment:
nit: since we are adding two more environments (3.10 + 3.11) in CIs, probably we can remove two existing environments (e.g., one Python 3.7 and one Python 3.8) to keep the number of CI pipelines.

Reply:
Done